Tests/EventListener/VersionListenerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Tests\EventListener;
+
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\View\ViewHandler;
+
+/**
+ * Version listener test.
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class VersionListenerTest extends \PHPUnit_Framework_TestCase
```

Comment:
Yes in part, I'm trying to understand how work the version listener to make this PR compatible.
Do you want me to split this change in another commit / PR ?